cognite/neat/_data_model/exporters/_table_exporter/exporter.py

```diff
@@ -49,34 +49,50 @@ def export(self, data_model: RequestSchema) -> DataModelTableType:
                     row[prop] = None
         return output
 
-    def as_excel(self, data_model: RequestSchema, file_path: Path, options: WorkbookOptions | None = None) -> None:
-        """Exports the data model as an Excel file.
+
+class DMSYamlExporter(DMSTableExporter):
+    """Exports DMS to YAML."""
+
+    def export(self, data_model: RequestSchema, file_path: Path) -> None:
+        """Exports the data model as a flat YAML file, which is identical to the spreadsheet representation
+
+        Args:
+            data_model (RequestSchema): The data model to export.
+            file_path (Path): The path to the YAML file to create.
+        """
+        table_format = self._export(data_model)
+        file_path.write_text(
+            yaml.safe_dump(table_format, sort_keys=False), encoding=self.ENCODING, newline=self.NEW_LINE
+        )
+
+
+class DMSExcelExporter(DMSTableExporter):
+    """Exports DMS to Excel file."""
+
+    def __init__(self, exclude_none: bool = False, options: WorkbookOptions | None = None) -> None:
+        super().__init__(exclude_none)
+        self._options = options or WorkbookOptions()
```

Comment:
For Excel, I think all columns should always be included, thus we hard code exclude None to False for Excel.
```suggestion
    def __init__(self, options: WorkbookOptions | None = None) -> None:
        super().__init__(exclude_none=False)
        self._options = options or WorkbookOptions()
```